cmd/utils.go

```diff
@@ -445,54 +461,69 @@ func getConfigLabels(config *RootCommandConfig) (map[string]string, error) {
 		return labels, projectConfigErr
 	}
 
-	labels["appsody.requested-stack"] = projectConfig.Stack
+	t := time.Now()
 
-	return labels, nil
-}
+	labels[ociKeyPrefix+"created"] = t.Format(time.RFC3339)
 
-func getGitLabels(config *RootCommandConfig) (map[string]string, error) {
-	gitInfo, err := GetGitInfo(config.Dryrun)
-	if err != nil {
-		return nil, err
+	var maintainersString string
+	for index, maintainer := range projectConfig.Maintainers {
+		maintainersString += maintainer.Name + ", " + maintainer.Email + ", " + maintainer.GithubID
```

Comment:
lets leave off the GithubID for now. just the maintainer name and email should be enough

Reply:
Also I think we should put the email in parenthesis and use commas only to separate multiple maintainers. 
For example:
`Neeraj Laad (neeraj.laad@gmail.com)`
and for multiple:
`Neeraj Laad (neeraj.laad@gmail.com), Kyle Christianson (christik@us.ibm.com)`

Reply:
Yeah, I prefer that - Good idea!